docs/mast/mast.rst

```diff
@@ -975,4 +975,8 @@ Reference/API
 
 .. automodapi:: astroquery.mast
     :no-inheritance-diagram:
+    :inherited-members:
+
+.. automodapi:: astroquery.mast.utils
```

Comment:
Got rid of the utils one. But needed to leave the `:inherited-members:` because otherwise all of the functions in `MastQueryWithLogin` like "login" and "session_info" aren't documented as they need to be and also building the docs fail (I think because I refer to those functions).

Reply:
yes, the comment was for `utils` only